kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala

```diff
@@ -109,6 +109,111 @@ private[v1] class SessionsResource extends ApiRequestContext {
     Response.ok().build()
   }
 
+  @POST
+  @Path("{sessionHandle}/executestatement")
+  def executeStatement(@PathParam("sessionHandle") sessionHandleStr: String,
+                       request: StatementRequest): OperationHandle = {
+    val sessionHandle = getSessionHandle(sessionHandleStr)
+    try {
+      backendService.executeStatement(sessionHandle, request.statement, request.runAsync,
+        request.queryTimeout)
+    } catch {
+      case NonFatal(_) =>
+        throw new NotFoundException(s"Error executing statement")
+    }
+  }
+
+  @POST
+  @Path("{sessionHandle}/gettypeinfo")
+  def getTypeInfo(@PathParam("sessionHandle") sessionHandleStr: String): OperationHandle = {
+    val sessionHandle = getSessionHandle(sessionHandleStr)
+    try {
+      backendService.getTypeInfo(sessionHandle)
+    } catch {
+      case NonFatal(_) =>
+        throw new NotFoundException(s"Error getting typeInfo")
+    }
+  }
+
+  @POST
+  @Path("{sessionHandle}/getcatalogs")
```

Comment:
I mean we should not contain verb e.g. `get` in the RESTful URI. The "R" in rest means resource, it is recommended to be a noun.

Reply:
What about `catalogs` or `catelogshandle`?

Reply:
Actually, this API creats a `getCatalogs` operation under a specific session. So, what about renaming to be `/sessions/{sessionHandle}/operations/catalogs`?